apps/aecore/lib/aecore/chain/chainstate.ex

```diff
@@ -67,8 +62,7 @@ defmodule Aecore.Chain.Chainstate do
       %Chainstate{} = new_chainstate ->
         {:ok,
          new_chainstate
-         |> Oracle.remove_expired_oracles(block_height)
-         |> Oracle.remove_expired_interaction_objects(block_height)}
+         |> Oracle.remove_expired_oracles(block_height)}
```

Comment:
if it is  only one command, the pipe is not needed.

are you sure `Oracle.remove_expired_interaction_objects` is not needed any more (maybe check with @d-velev)

Reply:
Function Oracle.remove_expired_oracles is removing expired oracles and interaction objects